test/torchaudio_unittest/prototype/transforms/batch_consistency_test.py

```diff
@@ -35,7 +35,7 @@ def test_batch_InverseBarkScale(self):
         n_barks = 32
         n_stft = 5
         bark_spec = torch.randn(3, 2, n_barks, 32) ** 2
-        transform = transforms.InverseMelScale(n_stft, n_barks)
+        transform = T.InverseBarkScale(n_stft, n_barks)
```

Comment:
@nateanl  I think this change caused style check to fail

https://app.circleci.com/pipelines/github/pytorch/audio/16245/workflows/cda9c3d4-4e35-4084-867d-e4e0ad4c4087/jobs/1209169